### ISSUE TITLE
How to add classification, for example, using cross entropy loss function to classify pictures 

### ISSUE THREAD

**Issue by Riting-Xia** [smatmo/EinsumNetworks_ICML20#1]
I have seen that ARt-SPN can be classified, and I plan to conduct the same classification in EinsumNetworks, but the training accuracy is very low, I can't find the reason, can you add its classification effect?  I look forward to receiving your reply. Thank you.
